src/components/modals/OrganizationInactiveModal.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <modal
+    :title="$t('~~Organization Inactive')"
+    data-testid="testVerificationTitleModal"
+    modal-classes="max-w-lg h-auto verification-modal"
+    :closeable="true"
+    @close="$emit('userLoggedOut')"
+  >
+    <div class="overflow-auto">
+      <div class="px-3">
+        <div
+          v-html="
+            $t(
+              `~~We're sorry. Your username and password are correct, but your organization is not currently active. Please contact helpdesk if you have any questions.`,
+            )
+          "
+        ></div>
+      </div>
+    </div>
+    <template #footer>
+      <div class="flex items-center justify-center py-2 border-t">
+        <base-button
+          data-testid="testVerificationLogoutButton"
+          :text="$t('actions.logout')"
+          :alt="$t('actions.logout')"
+          variant="solid"
+          class="px-6 p-3 logout-button"
+          :action="logout"
+        >
+          {{ $t('actions.logout') }}
+        </base-button>
+      </div>
+    </template>
+  </modal>
+</template>
+
+<script lang="ts">
+export default defineComponent({
+  name: 'OrganizationInactiveModal',
+  props: {
+    organization: {
+      type: Object,
+      default() {
+        return {};
+      },
+    },
+  },
+  emits: ['userLoggedOut'],
+  setup({ emit }) {
+    function logout() {
+      // Your logout logic here. This could involve emitting an event or
+      // calling some API endpoint or global function to perform the logout.
+      emit('userLoggedOut');
+    }
```

Comment:
@tabiodun gpt got this one wrong lmao.

`setup(props, {emit})` not `setup({emit})`

Reply:
Also the organization prop isnt even used :thinking:

Reply:
It’s funny because it actually didn’t. That was 100% my fault. 

It had used options API, and I switched to composition and forgot to make the change. It’s weird though because it still worked I thought when I tested it.

Reply:
lol, I assumed based on that comment, but makes sense.
Yeah it actually worked for me as well... but that might just be vite saving the day w/ macros